internal/cephfs/clone.go

```diff
@@ -33,6 +33,8 @@ const (
 	cephFSCloneInprogress = "in-progress"
 	// cephFSCloneComplete indicates that clone is in complete state.
 	cephFSCloneComplete = "complete"
+	// snapshotIsProtected indicates that snap protection state
+	snapshotIsProtected = "yes"
```

Comment:
this should probably be a bool?

Reply:
we have the return from the ceph backend in `string`, so its fine to keep it in string.